### PR TITLE
[BERT] [PyTorch] Data prep fix

### PR DESCRIPTION
**Pull request by sharathts**

### CODE REVIEW

File: PyTorch/LanguageModeling/BERT/data/bertPrep.py [hunk 1]
```diff
@@ -80,7 +84,7 @@ def main(args):
                 wikiextractor_command = path_to_wikiextractor_in_container + ' ' + directory_structure['download'] + '/' + args.dataset + '/wikicorpus_zh.xml ' + '-b 100M --processes ' + str(args.n_processes) + ' -o ' + directory_structure['extracted'] + '/' + args.dataset
                 print('WikiExtractor Command:', wikiextractor_command)
                 wikiextractor_process = subprocess.run(wikiextractor_command, shell=True, check=True)
-                #wikiextractor_process.communicate()
+                [[]]#wikiextractor_process.communicate()
```
Comment (nvcforster) [inline PyTorch/LanguageModeling/BERT/data/bertPrep.py]: Why add the [[]] ?

File: PyTorch/LanguageModeling/BERT/data/bertPrep.py [hunk 2]
```diff
@@ -127,15 +130,15 @@ def main(args):
     elif args.action == 'create_tfrecord_files':
         assert False, 'TFrecord creation not supported in this PyTorch model example release.' \
                       ''
-        if not os.path.exists(directory_structure['tfrecord']):
```
Comment (nvcforster) [inline PyTorch/LanguageModeling/BERT/data/bertPrep.py]: Please make sure the directory is created before the subdirectory

Comment (swethmandava) [reply PyTorch/LanguageModeling/BERT/data/bertPrep.py]: os.makedirs can create recursively, should be ok?

Comment (nvcforster) [reply PyTorch/LanguageModeling/BERT/data/bertPrep.py]: Right, I was thinking mkdir(). Should be fine

**Pull request merged**